java/gandiva/pom.xml

```diff
@@ -26,7 +26,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <protobuf.version>2.5.0</protobuf.version>
+        <protobuf.version>2.6.1</protobuf.version>
```

Comment:
sorry, missed this earlier. This could impact Dremio because it uses a different protobuf version. Can you please move the protobuf version change to a different PR ?

Reply:
sounds good..reverting this back.